ansible/resources/docker_sc4s.service

```diff
@@ -8,7 +8,7 @@ Requires=docker.service
 WantedBy=multi-user.target
 
 [Service]
-Environment="SC4S_IMAGE=ghcr.io/splunk/splunk-connect-for-syslog/container2:2"
+Environment="SC4S_IMAGE=ghcr.io/splunk/splunk-connect-for-syslog/container3:latest"
```

Comment:
Probably we need use "playbook variables" for that and substitude version using Jinja, I think it should be configurable.

Reply:
good point